labels.mss

```diff
@@ -533,12 +533,14 @@
      marker-max-error: 0.5;
      marker-spacing: 100;
      marker-fill: #777777;
-     [oneway != 'yes'][oneway_bicycle = 'yes'] {
-       marker-fill: #00f;
-     }
-     [highway='cycleway'] {
+     [highway='cycleway'],
+     [highway='path'] {
```

Comment:
It might be better to have an `is_primary_bike` value exported in the SQL output and filter on this one. This one, we could also account for some `bicycle=designated` cases or roads without motor traffic.

Might also be better in terms of Mapnik XML file size / etc.

Reply:
_is_primary_bike_ for what tags combination ?